zookeeper-server/src/main/java/org/apache/zookeeper/server/backup/BackupStatus.java

```diff
@@ -133,7 +135,9 @@ public synchronized void update(BackupPoint backupPoint) throws IOException {
     if (!statusFile.exists()) {
       LOG.info("BackupStatus::update(): BackupStatus file doesn't exist. Creating file at path: "
           + statusFile.getAbsolutePath());
-      statusFile.createNewFile();
+       if (!statusFile.createNewFile()) {
+         LOG.warn("Failed to delete file :{}", statusFile);
```

Comment:
nit: use `: {}` (space after colon) instead of ` :{}` (space before colon) as the former is the more widely followed format.